lms/djangoapps/shoppingcart/processors/CyberSource2.py

```diff
@@ -520,6 +525,16 @@ def _get_processor_exception_html(exception):
                 email=payment_support_email
             )
         )
+    elif isinstance(exception, CCProcessorUserDeclined):
+        return _format_error_html(
+            _(
+                u"Sorry! Our payment processor sent us back a message saying that transaction is declined. "
+                u"The items in your shopping cart will exist for future purchase. "
+                u"If you feel that this is in error, please contact us with payment-specific questions at {email}."
```

Comment:
Small grammar error here: it should be "sent us back a message saying that the transaction was declined"

@srpearce when you get a chance, could you please review this error message?  It's similar to the error message for when a user cancels the transaction.